README.md

```diff
@@ -4,6 +4,37 @@ Overview
 This repository contain documentation and scripts that aim to help PC Engines
 apuX platform users and developers to customize firmware to their needs.
 
+Versioning scheme change
+------------------------
+
+Since version `v4.8.0.1` we changed versioning scheme skipping coreboot `v4.7`
+release.
+
+# Why we changed versioning scheme?
+
+In recent version coreboot community introduced tag `v4.8.1` this breaks our
+previous versioning scheme which was `v4.6.z`, where `z` was PC Engines fork
+patch number. Because 3rd digit was already taken by coreboot tag this breaks
+our versioning scheme. As a result we start to use new versioning scheme
+`v4.8.0.w`, where `w` will be PC Engines fork patch number as always for each
+new release counted from 0.
```

Comment:
With this text, I would have expected first release from `v4.8` to have `w == 0`

Reply:
@kmalkki yes and that was my initial idea to have `0` release that is a pure merge of last PC Engines release with the new branch, but it happened that to make `v4.8` work after merging `v4.6.9` I added modifications what breaks that idea. So the questions is, what is better counting from `0` or from `1`? @miczyg1 ? We can of course push `v4.8.0.0` release on merge commit, but this would not be functional code.

Reply:
@pietrushnic there is no point in releasing pure v4.8.0.0 because nobody would use it (no working features). IMHO it is a good approach of starting with w==1, because its our first patch on 4.8.0 which enables all features and allow to use apu boards to its "fullest potential".

Reply:
@miczyg1 question is if `4.9.0.0` would be functional? I think it much more likely. Also is that mean we counting from 0 or from 1?

Reply:
@pietrushnic since we have a new release procedure based on syncing with mainline, 4.9.0.0 would be functional and each next `4.10.0.0` etc.  I really feel like starting from w==1, as it indicates our first patch. If I were a common user, seeing one more non-significant zero at the end would like nothing has changed. What I am saying may be a wrong idea, but we already started with w==1.